.env.example

```
@@ -0,0 +1,4 @@
+# Deployment used by `npx convex dev`
+CONVEX_DEPLOYMENT=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)